independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java

```diff
@@ -240,7 +240,7 @@ public InputStream createInputStream() {
             context.getBody().getBytes(data);
             return new ByteArrayInputStream(data);
         }
-        return new VertxInputStream(context, 10000, this);
+        return new VertxInputStream(context, getDeployment().getRuntimeConfiguration().readTimeout().toMillis(), this);
```

Comment:
This was already being used when copying an existing input stream here: https://github.com/quarkusio/quarkus/blob/fe1f9554fd2c49462acaf4a78e2d809a6bc22a82/independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java#L232